src/xpk/main.py

```diff
@@ -61,6 +61,7 @@
 
 xpk_print('Starting xpk', flush=True)
 main_args = parser.parse_args()
+main_args.enable_ray_cluster = False
```

Comment:
we shouldn't have any args being set here

Reply:
Already discussed this w/ @Obliviour

Reply:
Maybe try: https://github.com/AI-Hypercomputer/xpk/blob/e860597c408939ff7e5670f931fc35d4eb82f125/src/xpk/parser/storage.py#L81 with default=False ? But not sure, you will have to verify it.

Maybe the good solution would be to add create-ray, the same as we have create-pathways now? It would keep commands consistent. @Obliviour wdyt? . I am opting for this solution: create create-ray command, so we have similar patterns for different types of clusters. Sorry for review taking so long, but I think we are on good path

Reply:
crate-pathways still has a deprecated flag that is exposed to the user, in addition to the subcommand, which is how the subcommand works afaict: https://github.com/AI-Hypercomputer/xpk/blob/main/src/xpk/parser/cluster.py#L91

But I'll test your suggestion out

Reply:
Oh I misunderstood - your suggestion is still adding a flag, which we don't want